backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/objcexport/StubRenderer.kt

```diff
@@ -6,9 +6,13 @@
 package org.jetbrains.kotlin.backend.konan.objcexport
 
 import org.jetbrains.kotlin.descriptors.ClassDescriptor
+import org.jetbrains.kotlin.descriptors.DeclarationDescriptor
+import org.jetbrains.kotlin.descriptors.DeclarationDescriptorWithSource
+import org.jetbrains.kotlin.backend.common.serialization.extractSerializedKdocString
+import org.jetbrains.kotlin.backend.common.serialization.metadata.findKDocString
 
 object StubRenderer {
-    fun render(stub: Stub<*>): List<String> = collect {
+    fun render(stub: Stub<*>, shouldExportKDoc: Boolean): List<String> = collect {
```

Comment:
1. This is in fact public API used externally, its signature should not be modified like this.
2. `Stub` rendering should not essentially depend on any parameters: `Stub` class is already the structured internal representation for header fragments, including comments. So it is `ObjCExportTranslator`'s responsibility to decide if kdocs should be added to framework comments.

Fixing 2 might be complicated at the moment, because there is no single code fragment which builds comments for `Stub`s.
So, as a trade-off (involving technical debt), we could keep `Stub`s rendering _privately_ depend on `shouldExportKDoc`, while keeping _public_ API intact.